### PR TITLE
update zh_TW

### PR DESCRIPTION
**Pull request by mffsuyen**

### CODE REVIEW

**Comment by williewillus** [comment]
you need to explain your changes instead of handing me a 6000 line diff

**Comment by mffsuyen** [comment]
what this is the zh_tw language file, because the last translater give up translate this mod, so I am come, I redo the translate file for this mod

**Comment by mffsuyen** [comment]
I do this translate 3 month, you want I redo, hope not

**Comment by mffsuyen** [comment]
Idk whats your mean, the vazkii had update him lang file in en_us, so i need update my, as you can see the old zh_tw file is make a year ago, so in now vazkii add so many line in lang file, i need update it

**Comment by mffsuyen** [comment]
as you can see, in the old translate file, the Botania name in traditional chinese still is Botania but it can translate to 植物魔法 so i translate to 植物魔法 etc

**Comment by williewillus** [comment]
not sure what you mean. if you look in the "files changed" tab, it shows that Botania is already translated to "植物魔法"

**Comment by yrsegal** [comment]
Part of the issue Williewillus has with this Pull Request is that the difference between the files is so huge it makes it nearly impossible to understand.

This is because, at some point in editing, you changed the characters at the end of each line. [This](https://github.com/Vazkii/Botania/pull/2833/files?utf8=%E2%9C%93&diff=unified&w=1) page can show the diff without those changes.

In the future, @mffsuyen, please do not modify the formatting of a file you are submitting a Pull Request for.

**Comment by mffsuyen** [comment]
mine is correct, in the traditional the period is 。 not . the old translater is wrong

**Comment by mffsuyen** [comment]
In the Traditional Chinese, period should be "。", NatsuAurashi did ".", so he is wrong

**Comment by mffsuyen** [comment]
@williewillus And the tab is 植物魔法 may be he do right some where, but at other place, some is still Botania

File: src/main/resources/assets/botania/lang/zh_tw.lang [hunk 1]
```diff
@@ -1,3742 +1,3815 @@
...
-item.banner.botania_lexicon.green=綠色辭典
-item.banner.botania_lexicon.brown=棕色辭典
-item.banner.botania_lexicon.blue=藍色辭典
-item.banner.botania_lexicon.purple=紫色辭典
-item.banner.botania_lexicon.cyan=青色辭典
-item.banner.botania_lexicon.silver=淺灰色辭典
-item.banner.botania_lexicon.gray=灰色辭典
-item.banner.botania_lexicon.pink=粉色辭典
-item.banner.botania_lexicon.lime=淺綠色辭典
-item.banner.botania_lexicon.yellow=黃
```
Comment (williewillus) [inline src/main/resources/assets/botania/lang/zh_tw.lang]: these are the same as english, so no need to include them here

Comment (williewillus) [inline src/main/resources/assets/botania/lang/zh_tw.lang]: all of these Japanese names are Vocaloid song names. Are you sure that the Chinese names you have are the official names for those Vocaloid songs?

Comment (williewillus) [inline src/main/resources/assets/botania/lang/zh_tw.lang]: this is still in english

Comment (mffsuyen) [reply src/main/resources/assets/botania/lang/zh_tw.lang]: so what it should be?

**Comment by mffsuyen** [comment]
I did not see real comment

File: src/main/resources/assets/botania/lang/zh_tw.lang (hunk 1 above)
Comment (mffsuyen) [reply src/main/resources/assets/botania/lang/zh_tw.lang]: can you give me the url in the file changed?

**Comment by mffsuyen** [comment]
@williewillus but why in new en_us lang file not have use old song name's annotation?

**Comment by mffsuyen** [comment]
I have changed it

**Comment by mffsuyen** [comment]
@williewillus ?

**Comment by mffsuyen** [comment]
hey

**Comment by williewillus** [comment]
@3TUSK i think you've just signed yourself up to review the rest of this ;p

**Comment by 3TUSK** [comment]
> *This was a mistake what I have done*

Welp sure I can take a look... but we need to deal with new-line first so that we can have a saner diff.

File: src/main/resources/assets/botania/lang/zh_tw.lang (hunk 1 above)
Comment (NatsuArashi) [inline src/main/resources/assets/botania/lang/zh_tw.lang]: "You do not need to copy this into your lang file"

Comment (NatsuArashi) [inline src/main/resources/assets/botania/lang/zh_tw.lang]: 標點符號請一致。

Comment (NatsuArashi) [inline src/main/resources/assets/botania/lang/zh_tw.lang]: 這是雛菊吧什麼是皺菊?

**Pull request closed without merging**